slick-testkit/src/main/scala/com/typesafe/slick/testkit/util/StandardTestDBs.scala

```diff
@@ -218,13 +227,26 @@ object StandardTestDBs {
     override def capabilities =
       super.capabilities - TestDB.capabilities.jdbcMetaGetIndexInfo - TestDB.capabilities.transactionIsolation
 
-    /* Only drop and recreate the user. This is much faster than dropping
-     * the tablespace. */
-    override def dropUserArtifacts(implicit session: profile.Backend#Session) = {
-      session.close()
-      val a = DBIO.sequence(Seq(drop(0), create(1), create(2)).map(s => sqlu"#$s"))
-      await(databaseFor("adminConn").run(a))
+    override def localTables(implicit ec: ExecutionContext): DBIO[Vector[String]] = {
+      val tableNames = profile.defaultTables.map(_.map(_.name.name)).map(_.toVector)
+      tableNames
     }
+
+    override def localSequences(implicit ec: ExecutionContext): DBIO[Vector[String]] = {
+      // user_sequences much quicker than going to metadata if you don't know the schema they are going to be in
+      sql"select sequence_Name from user_sequences".as[String]
+    }
+
+    override def dropUserArtifacts(implicit session: profile.Backend#Session) =
```

Comment:
Is this faster than recreating the user or is the change due to constraints in the test environment?